Patches needed to compile strace on ARM (EABI).

Already included in recent BR.

Index: buildroot/package/strace/strace-arm-eabi-fix.patch
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ buildroot/package/strace/strace-arm-eabi-fix.patch	2010-06-04 09:25:00.000000000 +0200
@@ -0,0 +1,27 @@
+From f5da8d0c38ea0eb660cb17de7aa62cda6899eb75 Mon Sep 17 00:00:00 2001
+From: ldv <ldv>
+Date: Thu, 1 Jan 2009 23:20:38 +0000
+Subject: [PATCH] 2008-11-13  Kirill A. Shutemov  <kirill@shutemov.name>
+
+	* linux/arm/syscallent.h: Fix build on ARM EABI which does not
+	provide syscalls socketcall and ipc.
+
+diff --git a/linux/arm/syscallent.h b/linux/arm/syscallent.h
+index 5ccef2f..edc3641 100644
+--- a/linux/arm/syscallent.h
++++ b/linux/arm/syscallent.h
+@@ -431,6 +431,7 @@
+ 	{ 5,	0,	printargs,		"SYS_398"	}, /* 398 */
+ 	{ 5,	0,	printargs,		"SYS_399"	}, /* 399 */
+ 
++#ifndef __ARM_EABI__
+ #if SYS_socket_subcall != 400
+  #error fix me
+ #endif
+@@ -481,3 +482,4 @@
+ 	{ 4,	TI,	sys_shmdt,		"shmdt"		}, /* 440 */
+ 	{ 4,	TI,	sys_shmget,		"shmget"	}, /* 441 */
+ 	{ 4,	TI,	sys_shmctl,		"shmctl"	}, /* 442 */
++#endif
+-- 
+1.6.1.3
Index: buildroot/package/strace/strace-arm-no-cachectl.patch
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ buildroot/package/strace/strace-arm-no-cachectl.patch	2010-06-04 09:25:19.000000000 +0200
@@ -0,0 +1,19 @@
+ARM defines SYS_cacheflush, but doesn't have <asm/cachectl.h>
+---
+ system.c |    2 ++
+ 1 file changed, 2 insertions(+)
+
+--- a/system.c
++++ b/system.c
+@@ -70,9 +70,11 @@
+ #include <linux/capability.h>
+ #endif
+ 
++#ifndef ARM
+ #ifdef SYS_cacheflush
+ #include <asm/cachectl.h>
+ #endif
++#endif
+ 
+ #ifdef HAVE_LINUX_USTNAME_H
+ #include <linux/utsname.h>
